released/packages/coq-hoare-tut/coq-hoare-tut.v8.11.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "kartiksinghal@gmail.com"
+homepage: "https://github.com/coq-community/hoare-tut"
+dev-repo: "git+https://github.com/coq-community/hoare-tut.git"
+bug-reports: "https://github.com/coq-community/hoare-tut/issues"
+license: "LGPL-3.0-or-later"
+
+synopsis: "A Tutorial on Reflecting in Coq the generation of Hoare proof obligations"
+description: """
+Hoare logics are "program logics" suitable for reasoning about
+imperative programs.
+This work is both an introduction to Hoare logic and a demo
+illustrating Coq nice features. It formalizes the generation of PO
+(proof obligations) in a Hoare logic for a very basic imperative
+programming language. It proves the soundness and the completeness of
+the PO generation both in partial and total correctness. At last, it
+examplifies on a very simple example (a GCD computation) how the PO
+generation can simplify concrete proofs. Coq is indeed able to compute
+PO on concrete programs: we say here that the generation of proof
+obligations is reflected in Coq. Technically, the PO generation is
+here performed through Dijkstra's weakest-precondition calculus.
+"""
+
+build: [make "-j%{jobs}%" ]
+install: [make "install"]
+depends: [
+  "coq"
+]
+
+tags: [
+  "category:Mathematics/Logic/See also"
+  "category:Computer Science/Semantics"
+  "category:Compilation/Semantics"
+  "keyword:Hoare logic"
+  "keyword:imperative program"
+  "keyword:weakest precondition"
+  "keyword:reflection"
+  "logpath:HoareTut"
+]
+authors: [
+  "Sylvain Boulmé"
+]
+url {
+  src: "https://github.com/coq-community/hoare-tut/archive/v8.11.1.tar.gz"
```

Comment:
I believe the dependency on Coq should mention a version. Given this URL, I imagine it is >= 8.11

Reply:
Thanks for catching this. I am asking this question at [Zulip chat](https://coq.zulipchat.com/#narrow/stream/237663-coq-community-devs.20.26.20users/topic/author.20in.20opam.20packages/near/199862776) as well. But do you think mentioning something such as `>=8.10` is going to be enough?